NthKey/Model/Configuration.storekit

```diff
@@ -0,0 +1,79 @@
+{
+  "products" : [
+
+  ],
+  "settings" : {
+    "_timeRate" : 0
+  },
+  "subscriptionGroups" : [
+    {
+      "id" : "5BC28C3C",
+      "localizations" : [
+
+      ],
+      "name" : "subscription_group",
+      "subscriptions" : [
+        {
+          "adHocOffers" : [
+            {
+              "internalID" : "F0D0AE94",
+              "offerID" : "",
+              "paymentMode" : "free",
```

Comment:
Let's drop the free trial; the user can already use testnet and signet for free.
(even if this is just for testing, it reduces the amount of code which is always nice)